### PR TITLE
IGMGuesses: tweaks.

### PR DESCRIPTION
**Pull request by ntejos**
These changes address Issue #35 and Issue #36.
Depends on linetools' PR #251.

### CODE REVIEW

File: pyigm/guis/igmguesses.py [hunk 1]
```diff
@@ -50,6 +50,8 @@
 COLORS = ['#0066FF','#339933','#CC3300','#660066','#FF9900','#B20047']
 zero_coord = SkyCoord(ra=0.*u.deg, dec=0.*u.deg)  # Coords
 
+import sys
+sys.setrecursionlimit(150)
```
Comment (profxj) [inline pyigm/guis/igmguesses.py]: Is this needed?

File: pyigm/guis/igmguesses.py [hunk 2]
```diff
@@ -259,29 +267,36 @@ def __init__(self, ispec, parent=None, previous_file=None,
...
-            self.llist['available'] = linelist.subset_lines(reset_data=True, subset=names)
+            self.llist['available'] = linelist.subset_lines(subset=names, reset_data=True, verbose=True,
+                                                            sort_by='as_given')
             self.llist['List'] = 'available'
-        else:
-            self.llist['List'] = 'Strong'
-
+        else: # This should never happen
+            raise ValueError('Something is odd in update_available_lines! Please debug.')
+        print('Done.')
```
Comment (profxj) [inline pyigm/guis/igmguesses.py:295]: Remove?

**Comment by profxj** [comment]
Just a few comments here.
Merge once linteools PR#251 is merged

File: pyigm/guis/igmguesses.py (hunk 1 above)
Comment (ntejos) [inline pyigm/guis/igmguesses.py]: Not needed, will remove it.

File: pyigm/guis/igmguesses.py (hunk 2 above)
Comment (ntejos) [inline pyigm/guis/igmguesses.py:295]: Ok.

**Comment by ntejos** [comment]
There are some errors, from Python 3 setup and also in test_dndx_rvir(). Do not think these have to do with this PR though, shall I marge? @profxj

**Comment by profxj** [comment]
Merging

**Pull request merged**